sync-online/index.js

```diff
@@ -0,0 +1,66 @@
+// Our textarea
+var textarea = document.querySelector('textarea');
+
+// Register the ServiceWorker
```

Comment:
I know all of this `init()` stuff looks odd but I *really* don't want the fetch to happen before the service worker has had a chance to work.